packages/usa-time-picker/src/index.js

```diff
@@ -62,7 +62,7 @@ const transformTimePicker = (el) => {
 
   const selectEl = document.createElement("select");
 
-  ["id", "name", "required", "aria-label", "aria-labelledby"].forEach(
+  ["id", "name", "required", "aria-label", "aria-labelledby", "disabled", "aria-disabled"].forEach(
```

Comment:
What are these attributes for? There are enough now where we might want to extract it into a `const`.